docs/graphql/manual/guides/deployment/kintohub-one-click.rst

```diff
@@ -0,0 +1,61 @@
+.. meta::
+   :description: Deploy Hasura GraphQL engine in one-click on KintoHub
+   :keywords: hasura, docs, guide, deployment, kintohub, one-click
+
+Hasura GraphQL engine in one-click on KintoHub
+==============================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+The Hasura GraphQL engine is available in one-click on KintoHub. It is packed with a private `Postgres <https://www.postgresql.org/>`__ instance and secured with a free and automatic HTTPS certificate using `Let's Encrypt <https://letsencrypt.org/>`__.
+
+Quickstart
+----------
+
+1. Deploy your private Hasura and Postgres Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Click the button below to deploy your Hasura GraphQL engine and Postgres instances in your private environment. (``Ctrl+Click`` to open in a new tab)
+
+.. image:: https://resources.kintohub.com/deploybutton.png
```

Comment:
I think we should add steps on how to actually start a deployment. After logging into KintoHub, it's not quite clear what to do, as there seem to be several options.

Reply:
You just need to click on the button.
If you are connected, it will automatically deploy hasura + postgres in your environment.
If you are not connected or do not have an account, then you must log in first and then it will automatically deploy the stack.

Reply:
When I'm logged in and click on the button in the documentation, I see this:

![image](https://user-images.githubusercontent.com/5722022/77412600-62fa3500-6dbe-11ea-8006-e06e96fc1ba6.png)

Then I intuitively click on the `+` button, and see this:

![image](https://user-images.githubusercontent.com/5722022/77412730-90df7980-6dbe-11ea-8147-7c0844501174.png)

I'm not sure what to chose now and nothing is mentioned about Hasura, so it's not quite clear how to proceed.

Do you know what I mean?

Reply:
I think some screenshots could really help here 🙂

Reply:
@marionschleifer out of curiosity, what browser/OS are you using when you clicked on the deploy button. We were unable to repo:

1) be logged in to kintohub
2) click on the deploy button

Expected: Deploy hasura instantly
Actual: Land on overview page.

-- We are getting the Expected results, but it sounds like you were getting the Actual?

Reply:
@marionschleifer out of curiosity, what browser/OS are you using when you clicked on the deploy button. We were unable to repo:

1) be logged in to kintohub
2) click on the deploy button

Expected: Deploy hasura instantly
Actual: Land on overview page.

-- We are getting the Expected results, but it sounds like you were getting the Actual?